spec/models/tasks/special_case_movement_shared_examples.rb

```diff
@@ -5,9 +5,9 @@
 shared_examples "successful creation" do
   it "should create the SCM task and JudgeAssign task" do
     expect { subject }.not_to raise_error
-    scm_task =  appeal.tasks.where(type: described_class.name).first
+    scm_task =  appeal.tasks.of_type(described_class.name).first
```

Comment:
Should this be this?
```suggestion
    scm_task =  appeal.tasks.of_type(:described_class).first
```

Reply:
No. That doesn't work because `:described_class` would be converted to `"described_class"` which is not one of the task types.